Doc/library/ssl.rst

```diff
@@ -2686,8 +2686,8 @@ disabled by default.
 ::
 
    >>> client_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
-   >>> client_context.minimum_version = ssl.TLSVersion.TLSv1_3
-   >>> client_context.maximum_version = ssl.TLSVersion.TLSv1_3
+   >>> client_context.minimum_version = ssl.TLSVersion.TLSv1_2
+   >>> client_context.maximum_version = ssl.TLSVersion.MAXIMUM_VERSION
```

Comment:
The correct constant is `ssl.TLSVersion.MAXIMUM_SUPPORTED`

```suggestion
   >>> client_context.maximum_version = ssl.TLSVersion.MAXIMUM_SUPPORTED
```

https://docs.python.org/3/library/ssl.html#ssl.TLSVersion.MINIMUM_SUPPORTED

I don't see any constant alias with _VERSION, only _SUPPORTED according to:
https://github.com/python/cpython/blob/5a4af3ab030a3f3e708ee83d7d4ca3cb2d5b7360/Modules/_ssl.c#L220-L236